src/controllers/LessonController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+namespace App\Controllers;
+use Psr\Container\ContainerInterface;
+session_start();
+
+class LessonController
+{
+   protected $container;
+   public function __construct(ContainerInterface $container) {
+     $this->container = $container;
+   }
+
+  public function addLesson($request, $response, $args) 
+  {
+    $data = $request->getParsedBody();
+    
+    $name = filter_var($data['lessonname'], FILTER_SANITIZE_STRING);
+    $category = filter_var($data['category'], FILTER_SANITIZE_STRING);
+    $totalPage = filter_var($data['noofpages'], FILTER_SANITIZE_STRING);
+    $pages = $data["arr"];
+    $this->container->logger->info("----");
+
+    $lesson = [];
+    $pageArray = json_decode($pages, true);
+    for($i=0; $i<$totalPage; $i++){
```

Comment:
You can write this better as 
foreach($pageArray as $page){
array_push($lesson, $page);
}